python/cugraph/structure/graph.py

```diff
@@ -653,32 +653,6 @@ def from_dask_cudf_edgelist(
         self.destination_columns = destination
         self.store_tranposed = None
 
-    def compute_local_data(self, by, load_balance=True):
```

Comment:
Isn't there a unit test (`test_mg_utility.py`) that references this function?  Probably want to delete that as well (although CI skips the test because it's only single GPU)

Reply:
removed